composer.json

```diff
@@ -3,7 +3,7 @@
     "type": "neos-plugin",
     "name": "gesagtgetan/krakenoptimizer",
     "require": {
-        "neos/neos": "^3.0 || ^4.0",
+        "neos/neos": "*",
```

Comment:
guess this slipped in here for testing purposes, the next major of this plugin will require 4.3 (for the new ``thumbnail persisted`` signal and maybe more)

Reply:
Yes, I need to do this otherwise my composer goes crazy. Please have an eye on that when we are coming to a release-ready version.